docs/09-security/contingency-plan.md

```diff
@@ -96,15 +96,15 @@ CivicActions managed systems often depend on several external services. In the e
 - **Status:** <https://status.github.com/>
 - **Status:** <https://twitter.com/githubstatus>
 
-If GitHub becomes unavailable, DKAN/HDG will continue to operate in its current state. The
+If GitHub becomes unavailable, systems will continue to operate in its current state. The
 disruption would only impact the team's ability to update code on the instances.
 
 #### GitLab
 
-- **Service:** <https://git.civicactions.net/dsca/>
+- **Service:** <https://git.civicactions.net/>
 - **Status:** <https://app.statuscake.com/AllStatus.php?tid=1702974>
 
-If GitLab becomes unavailable, GlobalNET will continue to operate in its current state. The disruption would only impact the team's ability to update code on the instances.
+If GitLab becomes unavailable, systems will continue to operate in its current state. The disruption would only impact the team's ability to update code on the instances.
```

Comment:
From a company contingency perspective Gitlab is pretty critical - while it doesn't affect clients services, it does affect the teams ability to work. Not sure if/how to address this here?

Reply:
Increased note of impact.